nmma/em/analysis.py

```diff
@@ -744,6 +744,8 @@ def analysis(args):
 
     if args.bestfit:
         bestfit_to_write = bestfit_params.copy()
+        bestfit_to_write["ln_bayes_factor"] = round(result.log_bayes_factor, 3)
+        bestfit_to_write["ln_bayes_factor_err"] = round(result.log_evidence_err, 3)
```

Comment:
any particular reason for rounding? We don't round the other parameters stored in the json

Reply:
it was rounded upto 3 places in Bilby, so just adopting that :)

Reply:
I'd probably just leave it as the raw number, that way it can be rounded as needed